test/testlib/minikube_utilities.go

```diff
@@ -76,6 +77,17 @@ func standardizeSpaces(s string) string {
 	return strings.Join(strings.Fields(s), " ")
 }
 
+func RemoveEmptyLines(s string) string {
+	regex, err := regexp.Compile("(\r|\r\n|\n){2,}")
+    if err != nil {
+        return s
+    }
+    s = regex.ReplaceAllString(s, "\n")
+	s = strings.TrimRight(s, "\n")
+	
+	return s
+}
+
```

Comment:
We already have `standardizeSpaces` in this file, is that sufficient?

Reply:
My understanding is that `standardizeSpaces ` replace multiple spaces with one. Here I needed solution which removes empty lines in a multiline text. To be honest I didn't try `standardizeSpaces`